.circleci/config.yml

```diff
@@ -27,7 +27,6 @@ commands:
           command: |
             sudo apt update
             sudo apt install -y openjdk-11-jdk-headless
-      - install-awscli
```

Comment:
If this step is no longer being used, the step should be removed as well

Reply:
Removing that apt install step.